pkg/webhooks/sdnmigration/sdnmigration.go

```diff
@@ -45,6 +46,12 @@ type NetworkConfigWebhook struct {
 
 // Authorized will determine if the request is allowed
 func (w *NetworkConfigWebhook) Authorized(request admissionctl.Request) admissionctl.Response {
+	//We are doing this Check to endure That hive can trigger the
+	//migration process. Please refer to OSD-21711 for full explenation.
```

Comment:
Comments should be a single space after `//`. Additionally there are typos and inconsistent casing in the comment, please review.